external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -237,237 +234,6 @@ class RandomForestHyperparameters:
     max_depth: Optional[int] = None
     min_samples_split: Union[int, float] = 2
     min_samples_leaf: Union[int, float] = 1
-    max_features: Union[Literal["auto", "sqrt", "log2"], int, float] = "auto"
+    max_features: Union[str, int, float] = "auto"
     max_samples: Optional[Union[int, float]] = None
     bootstrap: bool = True
-
-
-@dataclasses.dataclass
-class _ModelTrainingConfig:
-    """Convenience wrapper for model training parameters and file info
-
-    Attrs:
-        model_type: sklearn model type or keras model class to initialize
-        hyperparameters: arguments to pass to model class at initialization
-            time
-        input_variables: variables used as features
-        output_variables: variables to predict
-        batch_function: name of function from `fv3fit.batches` to use for
-            loading batched data
-        batch_kwargs: keyword arguments to pass to batch function
-        data_path: location of training data to be loaded by batch function
-        scaler_type: scaler to use for training
```

Comment:
I'm not sure but I can make an educated guess. Given that I did a backwards-compatible refactor of dense training to use DenseHyperparameters above, and DenseHyperparameters does not contain scaler type/kwargs, I don't think this scaler was ever used with DenseModel. We needed to do mass scaling of values for the random forest because there's no other way to manipulate the loss function, whereas for the dense model we could do mass weighting directly in the loss (I don't know if this was ever used).

Reply:
The mass scaler was only compatible with the sklearn training in the past, aside from maybe existing in a dev branch somewhere.